src/index.md

```diff
@@ -7,3 +7,8 @@ about the core specification, as well as many modality-specific extensions.
 To get started, [check out the introduction](01-introduction.md). If you'd like
 more information on how to adapt your own datasets to match the BIDS
 specification, we recommend exploring the [bids-specification starter kit](https://github.com/bids-standard/bids-starter-kit).
+
+If you are interested in getting occasional updates on BIDS 
+please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit)
```

Comment:
```suggestion
please consider signing up for our [mailing list](https://docs.google.com/forms/d/1ZLi5qRTuX11KGK7qIidSdZvznFoXAqr2wh6003okv-0/edit).
```

Does the mailing list not have a more standard subscription management? e.g., most listservs have a page where you can sign up and then go back and unsubscribe if you want. Using a Google Form makes it feel like it's going to just be a list in somebody's contacts, which is hard to get off of.

Reply:
at this time no. having a more standard subscription management can be good at this point before using it (e.g. newsletters)

May you have recommendations on one?

Reply:
```suggestion

```